root/programs/C#/Frameworks/Infrastructure/ServiceInterface/AsyncProcessingService/TMInProcessDefinition.xml

```diff
@@ -6,5 +6,5 @@
 ]>
 <!-- idの先頭には、数字を使用できない。 -->
 <TMD>
-  <Transmission id="AzureStrageCopy" assemblyName="AsyncProcess" className="AsyncProcess.ProcessCopyJob" />
+  <Transmission id="AAA" assemblyName="AsyncSvc_sample" className="AsyncSvc_sample.LayerB" />
```

Comment:
This is a sample.
In that case, the following modification is required.
https://github.com/OpenTouryoProject/OpenTouryo/blob/develop/root/programs/C%23/Samples/AsyncSvc_sample/TestAsyncSvc_Sample/Program.cs#L81